### DISCUSSION TITLE
Client Side error (ReactJS)

### DISCUSSION THREAD

**Discussion by pgrandinetti** [scalar/scalar#1003] [Q&A]
I run into the same error that was referenced at https://github.com/scalar/scalar/discussions/732
The difference is that my app is in ReactJS.
I tried the same fix suggested in the linked discussion, but didn't solve it. Can you suggest anything?

EDIT: The error only happens after building the site. It does not happen when running locally (`npm run start`).

**Comment by marclave** [comment]
@pgrandinetti would you be able to try out our latest react package?

https://www.npmjs.com/package/@scalar/api-reference-react

**Reply by pgrandinetti** [reply]
@marclave Sorry for the late reply. I fixed this a few weeks ago, using `overrides`  instead of `resolutions` that was suggested in https://github.com/scalar/scalar/discussions/732
